Src/Support/Google.Apis.Auth/OAuth2/ImpersonatedCredential.cs

```diff
@@ -49,11 +49,6 @@ public sealed class ImpersonatedCredential : ServiceCredential, IOidcTokenProvid
             /// </summary>
             public IEnumerable<string> DelegateAccounts { get; set; }
 
-            /// <summary>
-            /// Gets the scopes to request during the authorization grant. May be null or empty.
-            /// </summary>
-            public IEnumerable<string> Scopes { get; set; }
```

Comment:
Hmm. I don't know if this is a binary breaking change. I suspect it may be. I'll investigate.

Reply:
I've just tested this and it was okay. But I'd like to find some documentation beyond experimentation... will have a look.

Reply:
https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/breaking-change-rules.md#members states that "Moving a method onto a class higher in the hierarchy tree of the type from which it was removed" is allowed, but the equivalent for a field isn't.

I've validated that my test with a property *does* show the problem when it's a field instead, so I think we're okay.